hoodie-client/src/main/java/com/uber/hoodie/HoodieWriteClient.java

```diff
@@ -792,6 +786,45 @@ private void clean(String startCleanTime) throws HoodieIOException  {
     }
 
     /**
+     * Performs a compaction operation on a MergeOnRead type of dataset. Depending on the CompactionStrategy,
+     * this operation will re-write log files(Avro) into parquet files.
+     * WARNING: Compaction operation cannot be executed asynchronously. Please always use this serially
+     * before or after an insert/upsert action.
+     * @param compactionTime
+     * @throws IOException
+     */
+    public void compact(String compactionTime) throws IOException {
```

Comment:
lets call this something like `forceCompact()` to call to subtly highlight, this also happens automatically otherwise..